index.html

```diff
@@ -5,12 +5,73 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+    <title>Soraiya's Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cardo&family=Karla:wght@300&family=Lora&family=Vollkorn&family=Vollkorn+SC&display=swap"
```

Comment:
You'll loading quite a few font families here but I can only see 2 being used in your CSS. Bear in mind that loading all these fonts in will slow down your page load time, so it's best to only include what you need.